### DISCUSSION TITLE
'fluidsynth.h': No such file or directory

### DISCUSSION THREAD

**Discussion by LucianoTheWindowsFan** [vgmtrans/vgmtrans#267] [Q&A]
So I was compiling the original vgmtrans. There is a build issue tag on here, but that is only for the QT version, and I'm compiling the WTL version.

I got an error about fluidsynth.h. I got it on the same directory as the file it looks for.

I'm working on support for Sunsoft's variant of the N-SPC.

**Comment by sykhro** [comment]
WTL doesn't use fluidsynth, it shouldn't even be looking for that file. How are you building it?

**Comment by LucianoTheWindowsFan** [comment]
I'm using Visual Studio 2019 for Windows. Thanks for the help.

**Comment by LucianoTheWindowsFan** [comment]
Here's the files for the format.
[SunNSPC.zip](https://github.com/vgmtrans/vgmtrans/files/7582617/SunNSPC.zip)

**Comment by sykhro** [comment]
To build WTL, you need to open the .vcprojx file and just hit compile. However, please note that you don't need to compile it yourself: latest WTL binaries are always here https://ci.appveyor.com/project/sykhro/vgmtrans-ci/branch/master/artifacts
